admin_manual/release_notes.rst

```diff
@@ -11,6 +12,45 @@ Release Notes
 * :ref:`8.0_release_notes_label`
 * :ref:`7.0_release_notes_label`
 
+.. _10.0.3_release_notes_label:
+
+Changes in 10.0.3
+-----------------
+
+Dear ownCloud administrator, please find below the changes and known issues of ownCloud Server 10.0.3 that need your attention:
+
+**The full ownCloud Server 10.0.3 changelog can be found here: https://github.com/owncloud/core/blob/stable10/CHANGELOG.md**
+
+* It is now possible to directly upgrade from 8.2.11 to 10.0.3 in a single upgrade process.
+* Added occ command to list routes which can help administrators setting up network firewall rules.
+* 'occ upgrade' is now verbose by default.
+    * Administrators may need to adjust scripts for automated setup/upgrade procedures that rely on 'occ upgrade' outputs.
+
+* Reenabled medial search by default
+    * Enables partial search in sharing dialog autocompletion (e.g. a user wants to share with the user "Peter": Entering "pe" will find the user, entering "ter" will only find the user if the option is enabled)
+    * New default is set to enabled as there is no performance impact anymore due to the introduction of the user account table in ownCloud Server 10.0.1.
+    * Please check the setting. You need to disable it if the functionality is undesired.
+
+* All database columns that use the fileid have been changed to bigint (64-bits).
+    * For large instances it is therefore highly recommended to upgrade in order to avoid reaching limits.
+
+* **Removed "themes" folder TODO: ask Philipp if still possible to use themes folder**
+
+* Upgrade and Market app information
+    * Removed "appstoreenabled" setting from config.php. If you want to disable the app store / Marketplace integration, please disable the Market app.
+    * Added setting 'upgrade.automatic-app-update' to config.php to disable automatic app updates with 'occ upgrade' when Market app is enabled
+    * On upgrade from OC < 10 the Market app won't be enabled if "appstoreenabled"  was false in config.php.
+
+* Clustering: Better support of read only config file and apps folder
+* Default minimum desktop client version in config.php is now 2.2.4.
+
+**Known issues**
+
+* Setting up SFTP external storages with keypairs does not work.
```

Comment:
I suggest linking to the tickets so admins can decide to subscribe/follow and maybe comment

Reply:
done.
Added user_shibboleth federation issue without link because of private repo